user-access-handlers/src/main/java/no/unit/nva/handlers/models/ImpersonationRequest.java

```diff
@@ -6,10 +6,10 @@
 
 public class ImpersonationRequest {
 
-    protected static final String NATIONAL_IDENTITY_NUMBER_FIELD = "nin";
+    protected static final String NATIONAL_IDENTITY_NUMBER_FIELD = "nationalIdentityNumber";
 
     @JsonProperty(NATIONAL_IDENTITY_NUMBER_FIELD)
-    @JsonAlias("nin")
+    @JsonAlias("nationalIdentityNumber")
```

Comment:
You can just delete the alias as its the same as the property. 

Also the frontend have not implemented Impersonation so no need to change this?